Chronic Break: psifinil DeForest medkits for spacers & other disease sufferers

Ephemeralis opened the pull request:
## About The Pull Request

Have you ever played a spacer (quirk) on Icemoon and thought to yourself: "wow, these ondansetron patches the game gives me are absolutely fucking useless?" If so, you're not alone! Ondansetron is possibly the worst chem in the entire codebase, and it is better off gone for the good of mankind.

The hardworking chemists at DeForest have had a solution for spacer-related gravity sickness and other forms of chronic sickness, stuttering or sleepiness this whole time: the mighty psifinil pen. And as the pen is mightier than the sword, I've decided to bundle up the following changes:

- A new civil defense medical kit variant titled the "civil defense symptom support kit" has been added. It is filled with *four* psifinil pens (which are 10u modafinil, 10u psicodine and 5u leporazine) perfectly positioned towards helping crew with conditions that involve jitteriness, stuttering, drowsiness, confusion, disgust, and sudden inexplicable swings in temperature.
- The civil defense symptom support kit (SSK from now on) is available at cargo for half the price of an ordinary civil defense kit given it only contains 1 type of autoinjector and doesn't actually heal anything.
- The SSK is also available as a medical department order, again at half price compared to a CDK.
- Spacers now spawn in with a free SSK instead of a USELESS bottle of ondansetron patches.

Astute chemical enthusiasts may note that modafinil is the "funny always-adjusting overdose" drug, a feature that tends to manifest itself when *used often across the course of a shift*. People playing temperature sensitive characters may appreciate a quick and themely fix to accidental exposure in the form of an autoinjector instead of mainlining a chocolate drink (did you know coco is almost as good as leporazine?).

## How This Contributes To The Nova Sector Roleplay Experience

This gives characters with mechanical chronic illnesses the means to reliably find some kind of symptomatic relief from their conditions if a chemist willing to make psicodine or modafinil is not available. On top of that, it eliminates the offensively useless stop-gap meds given to people with the Spacer quirk that were probably putting them off from playing the quirk entirely.

Even better, it continues the DeForest trend of Side Effects(tm) by introducing potentially large groups of people to the funny wonder that is modafinil's random overdose threshold! Medbay characters, rejoice!

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
![dreamseeker_ggdAVaXcoH](https://github.com/NovaSector/NovaSector/assets/966289/60b416ee-7ec8-4ab8-b5f0-da6997cc4a74)

</details>

## Changelog

:cl: yooriss
add: A new variant of the DeForest civil defense medical kit is now available: the civil defense symptom support kit. Loaded with four pens of psifinil, this little thing'll fit in your pocket and give your characters guaranteed relief from sleepiness, stuttering and sickliness. Some side effects may occur. Consult your doctor before use. This new kit is available cheaply from cargo and from medical's departmental orders, too.
balance: People with the Spacer quirk now start with a civil defense symptom support kit instead of a bottle of atrociously, unbelievably useless ondansetron patches.
/:cl:

Stalkeros2 commented:
make it blue like stalker military medkit pls

OrionTheFox commented on code/datums/quirks/positive_quirks/spacer.dm:
```diff
@@ -13,7 +13,7 @@
 	quirk_flags = QUIRK_CHANGES_APPEARANCE //NOVA EDIT CHANGE - ORIGINAL: quirk_flags = QUIRK_HUMAN_ONLY|QUIRK_CHANGES_APPEARANCE
 	medical_record_text = "Patient is well-adapted to non-terrestrial environments."
 	mail_goodies = list(
-		/obj/item/storage/pill_bottle/ondansetron,
```
Rather than non-modularly overwriting the TG code, why not leave all the TG stuff as-is and just supply a pen in the modular code using the same post_add proc? It would give them both items, sure, but it's more modular

Ephemeralis replied:
I really hate ondanestron.

OrionTheFox commented:
I like this concept though, it reminds me of the bone-/muscle-strengthening injectors in Ostranauts that serve a similar purpose. Something thats extremely fitting to the setting but seemingly forgotten by most. :+1:

Ephemeralis commented:
> Modafin and synaptizine is the medication given for players using the narcolepsy quirk yet you don't mention this quirk at all.

Is this a suggestion to change over the offered medication for narcolepsy as well? I haven't ever really played with the quirk so it never crossed my mind, honestly.

Ephemeralis commented:
> I think it would make sense to include the pill bottle narcolepsy quirkhavers get in the medkit, too. And then give both quirkhavers the kit.

Gone ahead and done this! I gave them the funny DeForest-like name of 'alifil' and modified the SSK to hold one 5-pill container of the drugs, and changed narcolepsy quirk to start them off with an SSK similar to Spacer.

kittysmooch commented:
any chance of getting a synth-chem equivalent of this for people running these quirks on synthetic characters?
